resources/scripts/ttx_diff.py

```diff
@@ -362,6 +362,22 @@ def sort_fontmake_feature_lookups(ttx):
         for (i, lookup_index) in enumerate(has_value):
             lookup_index.attrib['value'] = str(values[i])
 
+LOOKUPS_TO_SKIP = set([2, 4, 5, 6]) # pairpos, markbase, marklig, markmark
+
+def remove_mark_and_kern_lookups(ttx):
+    gpos = ttx.find("GPOS")
+    if gpos is None:
+        return
+    for lookup in gpos.xpath("//Lookup"):
+        lookup_type = int(lookup.find("LookupType").attrib["value"])
```

Comment:
nit: `attrib.get("value", 0)` so a lookup type with no value doesn't blow you up.

Reply:
this field should always exist and have a value; if it doesn't I would prefer to know about that and figure out why...